src/AstroBase.jl

```diff
@@ -1,5 +1,28 @@
 module AstroBase
 
-# package code goes here
+export earth_rotation_angle, anp
+const J2000 = 2451545.0
 
-end # module
+"""
+    earth_rotation_angle(jd1, jd2)
```

Comment:
Have another look at the recommended docstring format:
- First line: Function signature (indented)
- Second line: empty
- Third line and following: Explanation etc.